doc/api/readline.md

```diff
@@ -363,9 +402,13 @@ asynchronous iteration may result in missed lines.
 ### `rl.line`
 <!-- YAML
 added: v0.1.98
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/33676
+    description: Value will always be a string, never undefined.
 -->
 
-* {string|undefined}
+* {string}
```

Comment:
This probably makes this change semver-major, any reason this is in the same PR?

Reply:
I mentioned this in the original comment.
> Also an initial empty string was set for this.line since the
cursor methods fail if line is not initialized. This happened
in the tests when a question is asked and canceled before any
input has been received which would set line to a string. This
is due to questionCancel calling clearline.

To me this seemed like a bug but maybe we can find some other way around this if it is a too big change. But you have to help me with how you would like this solved in that case.